rosdep/python.yaml

```
@@ -6729,6 +6729,12 @@ python3-netifaces:
   openembedded: [python3-netifaces@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-netifaces']
   ubuntu: [python3-netifaces]
+python3-networkmanager:
+  debian:
+    buster: [python3-networkmanager]
+    jessie: [python3-networkmanager]
+    stretch: [python3-networkmanager]
+  ubuntu: [python3-networkmanager]
```

The automated checks have suggested a change that looks correct:
> Key 'python3-networkmanager' might be satisifed by fedora package named 'python3-networkmanager': https://src.fedoraproject.org/rpms/python-networkmanager#bodhi_updates

```suggestion
  fedora: [python3-networkmanager]
  ubuntu: [python3-networkmanager]
```

---

@cottsay Thank you for the review!
Just a double check. The link refers to `python-networkmanager` ( not a `python3-networkmanager` ). Is it still ok to commit the suggestion?
(I'm not a user of fedora and I'm not sure the name convention of fedora packages.)

---

@furushchev I think there is one source rpm `python-networkmanager`, but it is used to create a [`noarch` RPM called `python3-networkmanager`](https://koji.fedoraproject.org/koji/buildinfo?buildID=1754885). The `noarch` one is what's used to install the software. (@cottsay please correct me if I'm wrong)

---

This is correct. The binary package name is `python3-networkmanager`. Thanks for checking.

---

@cottsay @sloretz Thank you for double checking! I updated the commit.